### ISSUE TITLE
Deleting a report thow an issue

### ISSUE THREAD

**Issue by clrh** [widestage/widestage#70]
Reproduce:
* Delete a report
* Enter "DELETE"  "Are you sure you want to delete the report: xxx"
* Validate
* Red popup "Error deleting items, no item have been deleted"
* The line is always on the window
* Go to another window then go back on the "reports", the line is gone

Expected
* the line disapear when I validate

**Comment by clrh** [comment]
The "response" in debug is 
```
)]}',
{"data":"U2FsdGVkX1+o8r2xl0bHRye1CaYL1obGXfSPxN1xDErO/vf64XtoOuhgJdrv9PiWQ/o6cr9t0QxlkNKvj098wHb5iiGOXmIQhCTa4f1kcRuoBUyqWmZKH0KzHg/n/hKt"}
```
Maybe the json is malformed ? Can I obtain more debug data ?
Regards.

**Comment by pahan35** [comment]
I've fixed the issue in PR #83. When the author accept it, problem will be resolved.
